src/modules/Builder/features/ActivityItems/ItemConfiguration/ItemConfiguration.tsx

```diff
@@ -112,100 +111,99 @@ export const ItemConfiguration = () => {
     removeOptions();
   }, [selectedInputType]);
 
+  useEffect(() => {
+    !hasAlerts && setValue('alerts', []);
```

Comment:
pls do not set value if alerts checkbox is not checked because we need clear payload in form